agenthub/browsing_agent/response_parser.py

```diff
@@ -72,17 +72,33 @@ def check_condition(self, action_str: str) -> bool:
         return True
 
     def parse(self, action_str: str) -> Action:
-        thought = action_str.split('```')[0].strip()
-        action_str = action_str.split('```')[1].strip()
+        # parse the action string into browser actions and thought
```

Comment:
Could you please add a unit test to test this new behavior to demonstrate the expected behavior?